### PR TITLE
add property to specify the loggers where the SentryAppender is attached

### PR DESCRIPTION
**Pull request by marbon87**
## :scroll: Description
This pull-request adds the property sentry.loggers, which can be configured with a list of loggers, where the sentry appender is registered and defaults to the ROOT logger.

## :bulb: Motivation and Context
The current implementation of the spring-boot-starter-sentry registers the sentry appender only to the root logger.
If the logback configuration contains loggers, that are not  [additive](https://logback.qos.ch/manual/configuration.html#overrridingCumulativity), the log messages from children of the non addtive logger are not sent to sentry.

## :green_heart: How did you test it?
Added a new test to SentryLogbackAppenderAutoConfigurationTest

## :pencil: Checklist

- [x] I reviewed the submitted code
- [x] I added tests to verify the changes
- [ ] I updated the docs if needed
- [x] No breaking changes

### CODE REVIEW

**Comment by marbon87** [comment]
@adinauer the loggers are configured like this:

```
	<root level="ERROR">
		<appender-ref ref="APP_SYSTEM_OUT" />
		<appender-ref ref="SPRINGLOG" />
	</root>
	
	<logger name="de" level="OFF" additivity="false" />

	<logger name="de.foo.bar" level="WARN">
		<appender-ref ref="APP_INFO"/>
		<appender-ref ref="APP_WARNING"/>
		<appender-ref ref="APP_ERROR"/>
		<appender-ref ref="GELF"/>
		<appender-ref ref="SPRINGLOG" />
	</logger>
```

If i manually define the SentryAppender in logback.xml and use appender-ref for all loggers, i cannot use the comfortable configuration through spring boot by setting properties like sentry.context-tags.

**Comment by philipphofmann** [comment]
@adinauer, maybe you missed this PR somehow. Do we need more information from @marbon87, or can we review this PR?

**Comment by adinauer** [comment]
Nothing's missing as far as I know, I just didn't get around to doing the review yet - sorry.

File: sentry-spring-boot-starter/src/main/java/io/sentry/spring/boot/SentryProperties.java [hunk 1]
```diff
@@ -72,6 +77,15 @@ public void setLogging(@NotNull Logging logging) {
     this.logging = logging;
   }
 
+  @NotNull
+  public List<String> getLoggers() {
+    return loggers;
+  }
+
+  public void setLoggers(final @NotNull List<String> loggers) {
```
Comment (adinauer) [inline sentry-spring-boot-starter/src/main/java/io/sentry/spring/boot/SentryProperties.java:85]: I had to add the setter here to make the new test pass.

**Review by adinauer — COMMENTED** [review]
Thank you for the contribution @marbon87 and sorry for taking so long - we've been pretty busy lately. I've changed the PR a bit, can you please take another look and tell me what you think about them. If you're OK with them, I'll merge the PR.

**Comment by marbon87** [comment]
Sorry @adinauer, i overlooked your feedback. Your changes look good to me.

**Comment by marbon87** [comment]
Any updates on this?

**Review by adinauer — APPROVED** [review]
I've added a changelog entry and duplicated the changes into the jakarta module (for Spring Boot 3). I've also moved the config properties under `sentry.logging`. Thanks @marbon87 🚀

**Pull request merged**